po/de.po

```diff
@@ -64,6 +65,11 @@ msgid ""
 "in the File menu and rerun it later by selecting it from the menu Reports-"
 ">Preconfigured Reports."
 msgstr ""
+"Das bisherige <span class=\"gnucash\">GnuCash</span> Benutzer-definierte "
+"Berichte Verwaltung System wurde überarbeitet. Sie können einen Bericht "
+"speichern, durch Verwendung der Möglichkeiten im Dateimenü und den Bericht "
+"später wiederausführen durch auswählen aus dem Menü Reports-"
+">vorkonfigurierte Berichte."
```

Comment:
- "… <span class=\"gnucash\">GnuCash</span>-System zur Verwaltung von benutzerdefinierten Berichten …" wäre eine Möglichkeit, deutschen Satzbau umzusetzen.
- Auch der 2. Satz sollte umgestellt (oder aufgeteilt) werden.
- "Reports->" heißt in meinem Gnucash "Berichte"
- Im Programm sehe ich "Add the current report's configuration to the `Reports->Saved Report ". Guck mal, wie wir das übersetzt haben.